### ISSUE TITLE
bulker overwrites PS1

### ISSUE THREAD

**Issue by stolarczyk** [databio/bulker#50]
if I activate a crate, bulker creates a new prompt, which is awesome. But It also overwrites my previously set prompt configuration in `PS1`. Is it possible to prepend or append bulker prompt to whatever I have already set?

```
[mstolarczyk@MichalsMBP peppro](master): bulker-activate databio/peppro
Bulker config: /Users/mstolarczyk/bulker_config.yaml
Activating bulker crate: databio/peppro
databio/peppro|~/Uczelnia/UVA/code/peppro$
```

**Comment by nsheff** [comment]
Yes, it's possible to customize a shell prompt in the `shell_prompt` config option in your bulker config file.

However, this is broken at the moment, due to something broken about Yacman. See #46

**Comment by nsheff** [comment]
can you solve https://github.com/databio/yacman/issues/32 ?

**Comment by stolarczyk** [comment]
ok, will look into this

**Comment by nsheff** [comment]
Ok, after a long time I think the answer to this is to put your custom prompt in `shell_prompt` and just don't use quotes around it, or use singled quotes (better). And, use `\b` to refer to the bulker crate. So, for example add this to your bulker config:

```
  shell_prompt: '[\u@\h(\b) \W] $ '
```

**Comment by nsheff** [comment]
@stolarczyk can you see if this now solves your use case?

https://bulker.databio.org/en/dev/custom_prompts/

**Comment by nsheff** [comment]
you can also now use `-p` to not update prompts if you want to just keep using your default prompt.
